plugin/pkg/auth/authorizer/rbac/bootstrappolicy/controller_policy.go

```diff
@@ -249,6 +249,7 @@ func buildControllerRoles() ([]rbacv1.ClusterRole, []rbacv1.ClusterRoleBinding)
 			// used for pod eviction
 			rbacv1helpers.NewRule("update").Groups(legacyGroup).Resources("pods/status").RuleOrDie(),
 			rbacv1helpers.NewRule("list", "delete").Groups(legacyGroup).Resources("pods").RuleOrDie(),
+			rbacv1helpers.NewRule("get", "list").Groups(networkingGroup).Resources("clustercidrconfigs").RuleOrDie(),
```

Comment:
is this controller consuming this API?

Reply:
nevermind

https://github.com/kubernetes/kubernetes/blob/9187b343e17db2bf8f3470cd6e1ef7f661814c15/cmd/kube-controller-manager/app/core.go#L101-L110